src/main/java/nl/tudelft/contextproject/Main.java

```diff
@@ -229,10 +256,11 @@ protected void setupControlMappings() {
 			im.addMapping("Up", new KeyTrigger(KeyInput.KEY_W));
 			im.addMapping("Down", new KeyTrigger(KeyInput.KEY_S));
 			im.addMapping("Jump", new KeyTrigger(KeyInput.KEY_SPACE));
-			getInputManager().addMapping("Bomb", new KeyTrigger(KeyInput.KEY_Q));
-			getInputManager().addMapping("Pickup", new KeyTrigger(KeyInput.KEY_E));
+			im.addMapping("Bomb", new KeyTrigger(KeyInput.KEY_Q));
+			im.addMapping("Pickup", new KeyTrigger(KeyInput.KEY_E));
 		}
 
+		im.addMapping("Exit", new KeyTrigger(KeyInput.KEY_ESCAPE));

```

Comment:
This is intentional, and for multiple reasons:
* having one of the main controller buttons as Quit game button is inconvenient and confusing. When players try to figure out the controls, they will accidentally quit the game.
* the ESC key will always stop the game
* when mapping the Y on the controller to Exit, the game will exit upon startup.